server/webapp/WEB-INF/rails.new/app/presenters/api_v4/shared/config_origin/config_repo_origin_representer.rb

```diff
@@ -0,0 +1,38 @@
+##########################################################################
+# Copyright 2017 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV4
+  module Shared
```

Comment:
I don't see why this should be in shared. As far as I know, there is no other api that will make use of this representer in the immediate future.